tests/unit/core/statistics/test_constraints.py

```diff
@@ -67,6 +67,45 @@ def test_value_constraints(df_lending_club, local_config_path):
     assert report[1][1][0][2] == 50
 
 
+def test_value_constraints_pattern_match(df_lending_club, local_config_path):
+
+    regex_state_abbreviation = r"^[a-zA-Z]{2}$"
+    contains_state = ValueConstraint(Op.MATCH, regex_pattern=regex_state_abbreviation)
+
+    regex_date = r"^[a-zA-Z]{3}-[0-9]{4}$"
+    contains_date = ValueConstraint(Op.NOMATCH, regex_pattern=regex_date)
```

Comment:
Should this constraint variable should be name something like "not_contains_date" or "no_dates" since it is using Op.NOMATCH?